profiles/models.py

```diff
@@ -60,7 +60,17 @@ def __init__(self, country: Country, city: str, region: str = None) -> None:
         self.country = country
 
     def __composite_values__(self) -> Iterable[Optional[str]]:
-        return self.city, self.region, self.country
+        return self.country, self.city, self.region
+
+    def get_formatted(self) -> List[str]:
+        return [self.city, self.region, self.country.alpha2]
+
+    def get_components(self) -> Dict:
+        return {
+            "locality": [self.city],
+            "area": [self.region],
+            "country": self.country.name
```

Comment:
`get_formatted()` and `get_components()` are leaking JSON representation details into the model, we should keep them in the normalizer.

Reply:
Same with `Affiliation. get_name_list()`.

Reply:
OK agreed. Now all representation is in the normalize body:
```
@normalize.register(Affiliation)
def normalize_affiliation(affiliation: Affiliation) -> dict:
    return {
        "name": [affiliation.department, affiliation.organisation],
        "address": {
            "formatted": [
                affiliation.address.city,
                affiliation.address.region,
                affiliation.address.country.alpha2,
            ],
            "components": {
                "locality": [affiliation.address.city],
                "area": [affiliation.address.region],
                "country": affiliation.address.country.name
            }
        }
    }
```